src/styles/card.scss

```diff
@@ -0,0 +1,126 @@
+.catalog {
+  box-sizing: border-box;
+  display: grid;
+  grid-gap: 46px 48px;
+  justify-content: center;
+  grid-template-columns: repeat($column-count, 200px);
```

Comment:
This line isn't being used.
Just below it, you're declaring your media queries, and there are declarations for any width, thus those declarations are overwritting this one here.
The best approach you can do here is to remove the variable $column-count and replace it with `1`, then you can also remove the declaration for `@media (max-width: 487px)`.
The rest can remain the same